REPL.cpp

```diff
@@ -31,18 +30,30 @@ swift::FuncDecl *ConfigureFunctionLinkage(swift::SourceFile &src_file, std::uniq
 
             sil_module->lookUpFunction(sil_decl)->setLinkage(swift::SILLinkage::Public);
             Log(std::string("Set function ") + name_original + " (" + name_mangled + ") to public");
-
-            if(name_original == res_fn_name)
-                res_fn = fn;
         }
     }
     sil_module->lookUpFunction("main")->setLinkage(swift::SILLinkage::Private);
-    return res_fn;
 }
 
-llvm::Expected<std::unique_ptr<REPL>> REPL::Create()
+void REPL::RemoveRedeclarationsFromJIT(std::unique_ptr<llvm::Module> &llvm_module)
+{
+    SetCurrentLoggingArea(LoggingArea::SIL);
+
+    for(llvm::Function &fn : llvm_module->functions())
```

Comment:
clang-format?  Missing space after `for`.  Any reason for `fn` to not be `const`?